CustomTabBar/CustomTabBar.swift

```diff
@@ -69,12 +69,24 @@ public protocol CustomTabBarDelegate {
         }
     }
     
+    public var titleFont: UIFont = UIFont.systemFontOfSize(16) {
```

Comment:
well I can change it back to 10 or whatever, but this gives implementing controller access to the font. We have two different implementations of the CustomTabBar and they need different fonts